src/contentmanagerside.cpp

```diff
@@ -105,6 +107,10 @@ ContentManagerSide::~ContentManagerSide()
 void ContentManagerSide::setContentManager(ContentManager *contentManager)
 {
     mp_contentManager = contentManager;
+    const auto isLocal = mp_contentManager->isLocal();
+    const auto checkedButton = mp_ui->buttonGroup->button(isLocal);
```

Comment:
Same here :
`const auto checkedButton = mp_ui->buttonGroup->button((int)isLocal);`

(But I'm not sure it works with a enum, at least maybe not as simple)